subsys/net/lib/http/http_parser.c

```diff
@@ -1862,7 +1862,10 @@ int parser_execute(struct http_parser *parser,
 					__fallthrough;
```

Comment:
The commit message sounds non English. Could you change this to
`We should not ignore the body when the Transfer-Encoding is chunked.`